tests/PhpPact/Consumer/Matcher/Matchers/StringValueTest.php

```diff
@@ -21,6 +21,6 @@ protected function setUp(): void
     public function testSerialize(?string $value, string $json): void
     {
         $this->matcher = new StringValue($value);
-        $this->assertSame($json, json_encode($this->matcher));
+        $this->assertJsonStringEqualsJsonString($json, json_encode($this->matcher));
```

Comment:
no need for change, but see also: https://github.com/sebastianbergmann/phpunit/issues/4584

Reply:
noted. Thank you!